.github/workflows/labeler.yml

```diff
@@ -1,14 +1,46 @@
 name: "Pull Request Labeler"
 on:
-- pull_request_target
+- pull_request
 
 jobs:
   triage:
+    # We don't run the backporting for PRs from forks because those can't access "loki-github-bot" secrets in vault.
+    # We don't use GitHub actions app (secrets.GITHUB_TOKEN) because PRs created by the bot don't trigger CI.
+    # Also only run if the PR is merged, as an extra safe-guard.
+    if: ${{ ! github.event.pull_request.head.repo.fork && github.event.pull_request.merged == true }}  
```

Comment:
This workflow does not make sense if it cannot run on forks, and only when merged.